common/config/presets/c_base_node_c.toml

```diff
@@ -162,6 +162,8 @@ listener_self_liveness_check_interval = 15
 #rpc_max_simultaneous_sessions = 100
 # The maximum comms RPC sessions allowed per peer (default value = 10).
 #rpc_max_sessions_per_peer = 10
+# Cull the oldest peer RPC connection when the maximum number of sessions is reached (default value = true).
```

Comment:
```suggestion
# If true, and the maximum per peer RPC sessions is reached, the RPC server will close an old session and replace it with a new session. If false, the RPC server will reject the new session and preserve the older session. (default value = true).
```

Reply:
I have added some unit tests for this in the new commit